src/redux/Api/base.js

```diff
@@ -0,0 +1,9 @@
+/* const BASE_URL = 'https://rickandmortyapi.com/api';
+
+const connectApi = async (page = 1, type) => {
+  const response = await fetch(`${BASE_URL}/${type}?page=${page}`);
+  const data = await response.json();
+  return data;
+};
+
+export default connectApi;  */
```

Comment:
- [ ] Please remove the commented code to make your code cleaner and more readable